bot/cogs/bot.py

```diff
@@ -60,6 +62,23 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.content.count("\n") >= 3:
+            try:
+                tree = ast.parse(msg.content)
+                # Attempts to parse the message into an AST node.
+                # Invalid Python code will raise a SyntaxError.
+                if not all(isinstance(node, ast.Expr) for node in tree.body):
+                    # we dont want multiple lines of single words,
+                    # they would be syntactically valid Python but could also be
+                    # just some random multiline text someone is sending.
+                    formatted = f"```python\n{msg.content}\n```"
+                    if len(formatted) > 2000:
+                        await msg.channel.send(formatted)
+                        await self.bot.delete_message(msg)
```

Comment:
This is still async discord.py

Reply:
What do you mean?

Reply:
for reference: it would be `await bot.delete()`

Reply:
No, `await msg.delete()`, addressed it in my requested changes.

Reply:
uuh @shadycake bot.delete() made snekchek to show an error